### ISSUE TITLE
[Screen reader]: Narrator focus does not sync with keyboard focus while navigating on the table.

### ISSUE THREAD

**Issue by kidokidozh** [microsoft/fluentui#18208]
### Environment Information

- **Package version(s)**: 7.163.0
- **OS**: Version 20H2(OS Build 19042.830)
- **Browser**: Edge (Chromium) DEV Build Version (91.0..825.0) (Official Build) (64-bit)

#### Actual behavior:

Keyboard focus does not sync with narrator focus while navigating on the table using table mode (ctrl+alt+arows key), due to this, not able to perform any action as check/uncheck the checkbox.

#### Expected behavior:

Keyboard focus  should be in sync with narrator focus while navigating on table using table mode (ctrl+alt+arows key).

**Comment by gouttierre** [comment]
@kidokidozh - thank you for reaching out to us.

@smhigley  - Would you be able to confirm if this is a regression, or if this behavior is an issue with s check/uncheck checkboxes?
cc @miroslavstastny
